src/main/java/racingcar/controller/RacingController.java

```diff
@@ -0,0 +1,68 @@
+package racingcar.controller;
+
+import racingcar.domain.Car;
+import racingcar.domain.Cars;
+import racingcar.dto.CarDto;
+import racingcar.service.ParseCarNames;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+import java.util.List;
+
+public class RacingController {
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final ParseCarNames parseCarNames;
+
+    public RacingController(InputView inputView, OutputView outputView, ParseCarNames parseCarNames) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.parseCarNames = parseCarNames;
+    }
+
+    public void run() {
+        Cars cars = RacingStart();
+
+        moveCar(cars);
+
+        List<String> winner = getWinner(cars);
+
+        printWinner(winner);
+
+        inputView.closeConsole();
+    }
+
+    public void printWinner(List<String> winner){
+        outputView.printWinner(winner);
+    }
+
+    public List<String> getWinner(Cars cars) {
+        List<CarDto> winner = cars.findWinner();
+        return winner.stream().
+                map(CarDto::getName)
+                .toList();
+    }
+
+    public void moveCar(Cars cars) {
+        outputView.moveNumber();
+        int tryMove = inputView.getCarMoveNumber();
+        if(tryMove<=0){
+            throw new IllegalArgumentException();
+        }
+        for(int i=0; i<tryMove; i++){
+            List<CarDto> carDto = cars.moveCars();
+            outputView.printCarsInfo(carDto);
+        }
+    }
+
+    public Cars RacingStart() {
+        outputView.startRacing();
+        String msg = inputView.getCarsName();
+        List<String> carName = parseCarNames.parse(msg);
+        List<Car> cars = carName.stream()
+                .map(Car::from)
+                .toList();
+        return Cars.from(cars);
```

Comment:
변수명 메서드명이 전체적으로 알기 어려운 것 같아요...!!!
내부 구현까지 들어가지 않아도 역할이 드러나는 이름을 고민해보시면 어떨까요?

그리고 역할 분리가 더 필요해 보입니다 

1. 역할을 정확히 명시하면 좋을 것 같아요. `startRacing()`은 레이싱을 시작하는 것이 아니고 메시지를 출력하고 있네요.
2. `getXX()`는 의미가 모호하기도 하고 보통 getter에서 쓰이기 때문에 혼란이 올 수 있을 것 같아요
3. 리스트인 경우 복수형으로 쓰면 의미가 더 알기 쉬울 것 같습니다.
4. `msg` 처럼 축약형 이름을 사용하면 다른 개발자가 읽기 어려워집니다. `message`로 바꾼다고 해도 아직 모호한 것 같아요.

```suggestion
        outputView.printInputCarNamesAnnounce();
        String rawCarNames = inputView.readCarNames();
        List<String> carNames = parseCarNames.parse(rawCarNames);
        List<Car> cars = carNames.stream()
                .map(Car::from)
                .toList();
        return Cars.from(cars);
```